config/samples/bpfman.io_v1alpha1_bpfapplication.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: bpfman.io/v1alpha1
+kind: BpfApplication
+metadata:
+  labels:
+    app.kubernetes.io/name: bpfapplication
+  name: bpfapplication-sample
+spec:
+  # Select all nodes
+  nodeselector: {}
+  bytecode:
+    image:
+      url: quay.io/bpfman-bytecode/go-application-counter:latest
+  programs:
+    - type: Kprobe
+      kprobe:
+        bpffunctionname: kprobe_counter
+        func_name: try_to_wake_up
+        offset: 0
+        retprobe: false
```

Comment:
maybe we can inline so it can be something like
```yaml
    - type: Kprobe
        bpffunctionname: kprobe_counter
        func_name: try_to_wake_up
        offset: 0
        retprobe: false
```
I can give it a try and see if that will work ?

Reply:
No I'd rather get CEL you can leave it 👍